static/app/views/alerts/list/rules/alertRuleStatus.tsx

```diff
@@ -86,20 +89,26 @@ export default function AlertRuleStatus({rule}: Props) {
 
   return (
     <FlexCenter>
-      <IconArrow color={iconColor} direction={iconDirection} />
-      <TriggerText>
-        {`${thresholdTypeText} ${
-          rule.latestIncident || (!rule.latestIncident && !resolvedTrigger)
-            ? trigger?.alertThreshold?.toLocaleString()
-            : resolvedTrigger?.toLocaleString()
-        }`}
-        {getThresholdUnits(
-          rule.aggregate,
-          rule.comparisonDelta
-            ? AlertRuleComparisonType.CHANGE
-            : AlertRuleComparisonType.COUNT
-        )}
-      </TriggerText>
+      {rule.detectionType !== AlertRuleComparisonType.DYNAMIC ?? (
```

Comment:
nit: `??` (nullish coalescing) i think is mainly used when we want to default a `null` or undefined value.
I _think_ `&&` is prefered when you want to conditionally determine whether to render something or not.

Reply:
oops